contracts/Migrations.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity >=0.4.21 <0.6.0;
```

Comment:
how about using this package/contract ?
https://github.com/daostack/web3-transaction-batcher/blob/master/contracts/Batcher.sol#L4